stm32mp1.mk

```diff
@@ -0,0 +1,212 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+################################################################################
+override COMPILE_NS_USER   := 32
+override COMPILE_NS_KERNEL := 32
+override COMPILE_S_USER    := 32
+override COMPILE_S_KERNEL  := 32
+
+PLATFORM ?= stm32mp1-157C_DK2
+OPTEE_OS_PLATFORM := $(PLATFORM)
+
+include common.mk
+
+ifeq ($(PLATFORM),stm32mp1-157A_DK1)
+BREXT_FLAVOR		= STM32MP157A-DK1
+STM32MP1_DTS_BASENAME	= stm32mp157a-dk1
+else ifeq ($(PLATFORM),stm32mp1-157C_DK2)
+BREXT_FLAVOR		= STM32MP157C-DK2
+STM32MP1_DTS_BASENAME	= stm32mp157c-dk2
+else ifeq ($(PLATFORM),stm32mp1-157C_EV1)
+BREXT_FLAVOR		= STM32MP157C-EV1
+STM32MP1_DTS_BASENAME	= stm32mp157c-ev1
+else
+$(error Unknown PLATFORM $(PLATFORM))
+endif
+
+################################################################################
+# Binary images names
+################################################################################
+
+ARM_TF_BIN		:= tf-a-$(STM32MP1_DTS_BASENAME).stm32
+OPTEE_HEADER_BIN	:= tee-header_v2.stm32
+OPTEE_PAGER_BIN		:= tee-pager_v2.stm32
```

Comment:
Is it GitHub fooling me? (indentation)?

Reply:
Indentation is fine. Fuzzy in the github web rendering.